projects/packages/search/build/webpack.config.search-dashboard.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+const getBaseWebpackConfig = require( '@automattic/calypso-build/webpack.config.js' );
```

Comment:
Let's avoid adding new calypso-build dependencies. See what I did with Jetpack in #21581.